src/decoding.rs

```diff
@@ -84,7 +84,7 @@ pub fn decode(
     padding: u32,
     format: u8,
 ) -> Result<Vec<u8>, CarbonadoError> {
-    let format = Format::try_from(format)?;
+    let format = Format::from(format);
```

Comment:
Interesting you dropped "try_from"  to "from", isn't that a downgrade. 
Just wondering the logic for this.

Reply:
No, that's better. The library we're using can guarantee infallibility.